tests/scripts/pulp_ostree/test_modify_content.sh

```diff
@@ -64,20 +78,49 @@ expect_succ pulp ostree repository ref add --repository "cli_test_ostree_reposit
 
 # list content stored within the latest repository versions
 expect_succ pulp ostree repository ref list --repository "cli_test_ostree_repository1"
-expect_succ pulp ostree repository ref list --repository "cli_test_ostree_repository1" \
-  --type "ref"
+
+if [ "$CLI15" = "True" ]
+then
+  expect_succ pulp ostree repository ref --type "ref" list \
+    --repository "cli_test_ostree_repository1"
+else
+  expect_succ pulp ostree repository ref list --repository "cli_test_ostree_repository1" \
+    --type "ref"
+fi
```

Comment:
I don't quite understand this syntax. Is `...tory ref list ...` really only listing refs? What happens if you'd `...tory ref --type "commit" list ...`?